dwave/embedding/transforms.py

```diff
@@ -63,6 +159,11 @@ def embed_bqm(source_bqm, embedding, target_adjacency, chain_strength=1.0,
             target graph) in SPIN or BINARY space. Defaults to the
             :class:`.Vartype` of `source_bqm`.
 
+        embedded_structure (:class:`.EmbeddedStructure`, optional):
```

Comment:
Could we note just accept an `EmbeddedStructure` for the `embedding` arg? And then make `target_adjacency` optional, depending on the type of `embedding`.